File: ae00.c
#include<stdio.h>
int solution(int n, int i,int temp);
int main()
{
	int n,i=1,ans;
	scanf("%d",&n);
	ans = solution(n,i+1,n/i - i + 1);
	printf("%d",ans);
	return 0;
}
int solution(int n , int i , int temp)
{
	if (temp >=0) return temp + solution(n,i+1,n/i - i + 1);
	else return 0;
}

File: ae00.py
import sys
input_data = sys.stdin.readlines()
input_data = int(input_data[0].rstrip())
i = 1
ans = 0
while input_data/i - i + 1 >= 0:
	temp = input_data/i - i + 1
	ans += temp
	i += 1
print ans
